cmake/defaults/Options.cmake

```diff
@@ -24,6 +24,9 @@
 option(PXR_STRICT_BUILD_MODE "Turn on additional warnings. Enforce all warnings as errors." OFF)
 option(PXR_VALIDATE_GENERATED_CODE "Validate script generated code" OFF)
 option(PXR_HEADLESS_TEST_MODE "Disallow GUI based tests, useful for running under headless CI systems." OFF)
+option(PXR_OUTPUT_FAILED_TEST_DIFFS
+    "Copy failed diff/image diff tests into \${CMAKE_BINARY_DIR}/Testing/Failed-Diffs/<PXR_CTEST_RUN_ID>/\${TEST_NAME}. <PXR_CTEST_RUN_ID> is automatically set per ctest-invocation, or read from an environment variable of the same name if set."
+    OFF)
```

Comment:
Is there a particular reason we wouldn't want to have this on all the time? We think it make sense to turn this on all the time for simplicity?

Reply:
Sure, I'm all for that! The only argument against having it on by default is the it means the build directory might grow over time - each test run with a failed image diff will result in a new directory in `${CMAKE_BINARY_DIR}/Testing/Failed-Diffs`.  However, that's no different (and much less than) the filesystem usage growth that already occurs in the temp directories (it's just more easily visible / discoverable).  But I think the pros outweigh the cons - ideally you won't have failed tests too often, and only more "advanced" users are probably running the tests a lot anyway.

Reply:
Removed:
https://github.com/pmolodo/USD/commit/c24838f63735082c515e7dc8b99032397c949c72#diff-9164722da603880b3c8f0383cdf4f24c3fd29963e4da223c9d6e7cb9c4119e4aR780